Lecture Material/Week 02 - HTML, css, javascript; Python Variables, Expressions, Statements/02 - HTML, CSS, and Javascript.ipynb

```diff
@@ -296,9 +296,31 @@
    "source": [
     "### No Seriously, It's Really Weird:\n",
     "\n",
+    "Every language has it's peculiarities...\n",
```

Comment:
I would actually prefer to add this to the lesson on Numbers and Math (or maybe a later one when we talk about comparisons).  I'm not sure it fits here in that the data will be retained by the reader for when it really counts.

Reply:
makes sense. That is what I was wondering. I thought that was the spot that it fit best in the current lessons.